packages/api-client/src/CumulusApiClientError.ts

```diff
@@ -1,7 +1,12 @@
 export class CumulusApiClientError extends Error {
-  constructor(message: string) {
+  public statusCode: number | undefined;
+  public apiMessage: string | undefined;
+
+  constructor(message: string, statusCode: number, apiMessage: string | undefined) {
```

Comment:
This update and the update to the invoke method allows for programmatic inspection of the error without needing to parse the usertext message.    This is helpful if you're making a call to invoke the API but want to take different action depending on the api failure *without* modifying the throw/catch behavior. 

I'm open to thinking about explicitly passing in a *list* of status codes and parsing a response, but I think generally the try/catch style is better/more explicit than a bunch of if branch logic for a successful return.